cv_next/app/feed/components/feed.tsx

```diff
@@ -128,16 +130,22 @@ export default function Feed() {
           <TriggerPagination callbackTrigger={fetchCvsCallback} />
         </div>
         {!loadMore ? (
-          <div className="sticky bottom-5 z-10 flex justify-center">
-            <ReloadButton callback={forceReload}>Reload</ReloadButton>
+          <div className="text-center">
+            {cvs.length ? (
+              <div className="sticky bottom-5 z-10 flex justify-center">
+                <ReloadButton callback={forceReload}>Reload</ReloadButton>
+              </div>
+            ) : (
+              <p>No CVs found that matched your filters</p>
+            )}
           </div>
         ) : (
           <div className="z-10 flex justify-center">
             <ReactLoading
               type={"spinningBubbles"}
               color={"#000"}
               height={667}
-              width={375}
+              width={290}
```

Comment:
Are these pixel sizes? If so, can this be viewport-based values instead?

Reply:
Changed it so that it's based on percentage